osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/application/services/ActionMapperService.java

```diff
@@ -248,7 +249,7 @@ public BundleMessagesRequestDto mapAllActions(final BundleMessageRequest bundleM
                 if (mapper != null) {
 
                     if (action instanceof MbusActionRequest) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS134)

Reply:
Done